requirements.yml

```diff
@@ -87,6 +87,7 @@
 - name: APLpy
   version: '1.0'
   setup_options: '--offline'
+  python: '>2.6,<3.5.2'
```

Comment:
Probably not, since the CI files impose a lower limit of 2.7*. In general, though, yes, because there is a python 1.0 build on defaults...